liferay-workspace/tests/blade.samples.integration.test/src/testIntegration/java/com/liferay/blade/samples/test/BladeSamplesTest.java

```diff
@@ -109,24 +109,53 @@ public void testAllBladeSamples() throws Exception {
 		}
 
 		for (String sampleBundleFile : bladeSampleOutputFiles) {
-			String installBundleOutput = BladeCLIUtil.installBundle(
-				new File(sampleBundleFile));
-
-			String printFileName = new File(sampleBundleFile).getName();
-
-			bundleIDAllMap.put(installBundleOutput, printFileName);
-
-			try (Jar jar = new Jar(sampleBundleFile, sampleBundleFile)) {
-				if (jar.getManifest().getMainAttributes().getValue(
-						"Fragment-Host") == null) {
-
-					bundleIDStartMap.put(installBundleOutput, printFileName);
+			String printFileName;
+			String installBundleOutput;
+			
+			if (sampleBundleFile.endsWith(".war")) {
+				printFileName = new File(sampleBundleFile).getName();
+
+				String output = BladeCLIUtil.execute(
```

Comment:
Yes it includes the extensions. Everything worked okay, but do you think i should trim the extensions?